### DISCUSSION TITLE
what's  blocked_response_ttl

### DISCUSSION THREAD

**Discussion by Yumega** [AdguardTeam/AdGuardHome#7267] [Q&A]
blocked_response_ttl: 10 
set  blocked_response_ttl into 10 or 0?

**Comment by Dynamic5912** [comment]
Why would you use 0 ?

**Comment by ppfeufer** [comment]
> what's blocked_response_ttl?

https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration

> `blocked_response_ttl`: For how many seconds the clients should cache a filtered response. Low values are useful on LAN if you change filters very often, high values are useful to increase performance and save traffic.
